src/python/director/cameraview.py

```diff
@@ -132,6 +133,7 @@ def addImage(self, name):
         self.imageUtimes[name] = 0
         self.images[name] = image
         self.textures[name] = tex
+        self.imageRotations180[name] = None
```

Comment:
This could be a set instead of a dict.  If you prefer a dict of booleans, maybe initialize to False instead of None?